xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-ui/src/main/resources/CKEditor/EditSheet.xml

```diff
@@ -181,9 +181,11 @@ define('xwiki-ckeditor', [
     }
 
     var uploadDisabled = element.hasAttribute('data-upload-disabled');
+    var startupFocus = element.hasAttribute('data-startup-focus');
 
     var config = {
       filebrowserUploadUrl: uploadDisabled ? '' : getUploadURL(sourceDocument, 'filebrowser'),
+      startupFocus,
```

Comment:
shouldn't it be `startupFocus: startupFocus`? it's a key: value config object no?

Reply:
It's called "shorthand property name" see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#property_definitions (syntax sugar to reduce boilerplate).

Reply:
ok thanks for the information, I'm not sure I'd have changed it since IMO it's not helping much reading but that's fine :)